web-app/src/components/common/Errorbox/styled.js

```diff
@@ -1,18 +1,25 @@
 import styled from 'styled-components';
 
 export const ErrorContainer = styled.div`
+<<<<<<< HEAD
```

Comment:
Missed a wee conflict here

Reply:
Good spot... Don't know why this hasn't been picked up by my editor or Github 😕